pyterrier/__init__.py

```diff
@@ -63,9 +64,12 @@ def init(version=None, mem=None, packages=[], jvm_opts=[], redirect_io=True, log
      * If the `version` init kwarg is not set, Terrier will query MavenCentral to determine the latest Terrier release.
      * If `version` is set to `"snapshot"`, the latest .jar file build derived from the `Terrier Github repository <https://github.com/terrier-org/terrier-core/>`_ will be downloaded from `Jitpack <https://jitpack.io/>`_.
      * Otherwise the local (`~/.mvn`) and MavenCentral repositories are searched for the jar file at the given version.
-     
     In this way, the default setting is to download the latest release of Terrier from MavenCentral. The user is also able to use a locally installed copy in their private Maven repository, or track the latest build of Terrier from Jitpack.
     
+    If your running enviroment is offline. you can download the "terrier-assemblies-{your version}-jar-with-dependencies.jar" and "terrier-python-helper-{your helper version}.jar"
```

Comment:
Its enough to run pyterrier online, and then to set the versions for the offline scenario, right?

Reply:
Yes, you need to set the versions and helper_version so that you can run offline.